tests/iterator.rs

```diff
@@ -1,3 +1,5 @@
+#![cfg(not(windows))]
+
```

Comment:
AFAIK this is not needed any more, the condition is already on the `pub mod` in `lib.rs`.

Reply:
This isn't `src/iterator.rs`, but `tests/iterator.rs`.

Reply:
I see, oversight on my side :-). It makes sense.